docs/install/linux_source.rst

```diff
@@ -23,7 +23,8 @@ PyNE has the following dependencies:
 
 Optional Depenendencies:
    #. `MOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
-   #. `PyTAPS <https://pythonhosted.org/PyTAPS/index.html>`_
+   #. `pyMOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
```

Comment:
I wonder if this should be listed as a part of MOAB rather than its own line.  It is not really a separate dependency, and you need pyMOAB to use MOAB

Reply:
ping @bam241